lib/state/ui/reducer.ts

```diff
@@ -22,4 +26,42 @@ const visiblePanes = (state = defaultVisiblePanes, { type, show }) => {
   return state;
 };
 
-export default combineReducers({ filteredNotes, visiblePanes });
+const setSelectedNote = (state = null, action) => {
```

Comment:
in Redux fashion we'd rather name this reducer `selectedNote` or even just `note` since it describes a data type and the ways that data changes vs. describing a _setter_ which updates data.

we can see how each reducer acts as an independent OO actor governing its own state and responding to messages received from "the outside world" by means of the actions